experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -164,12 +182,13 @@ export class NodeSDK {
       });
     }
 
-    if (this._metricReader) {
+    if (this._meterProviderConfig?.reader) {
```

Comment:
I may not follow the point of checking the presence of the metric reader instead of checking the presence of the meter provider config here.

Would it be better to align the behavior with the tracer provider in this section that checks the presence of tracer provider config / meter provider config?

Reply:
@weyert I noticed that you marked this as resolved but I didn't find any changes on this. Do you think this is intended? If so would you mind elaborating on this?

Reply:
@legendecas You are right, looks like the push earlier. I have pushed my latest code

Reply:
It is still checking for the reader itself and not the config unless i'm mistaken?